api/nodes/views.py

```diff
@@ -293,11 +305,30 @@ def perform_create(self, serializer):
     # overrides BulkDestroyJSONAPIView
     def allow_bulk_destroy_resources(self, user, resource_list):
         """User must have admin permissions to delete nodes."""
+        if is_truthy(self.request.query_params.get('skip_uneditable', False)):
+            return any([node.has_permission(user, ADMIN) for node in resource_list])
         for node in resource_list:
             if not node.has_permission(user, ADMIN):
                 return False
         return True
 
+    def bulk_destroy_skip_uneditable(self, resource_object_list, user, object_type):
+        """
+        If skip_uneditable=True in query_params, skip the resources for which the user does not have
+        admin permissions and delete the remaining resources
+        """
+        allowed = []
+        skipped = []
+        for resource in resource_object_list:
+            if resource.has_permission(user, ADMIN):
+                allowed.append(resource)
+            else:
+                skipped.append({'id': resource._id, 'type': object_type})
+
+        if is_truthy(self.request.query_params.get('skip_uneditable', False)):
```

Comment:
You should probably do this check before the costly resource permission calculation.
```python
if not is_truthy(self.request.query_params.get('skip_uneditable', False)):
    return None

# build skipped / allowed dict```